docs/ffi.md

```diff
@@ -422,16 +417,16 @@ Since this documentation page is automatically generated on a machine without ac
 When defining our FFI wrapper for CPU, the function signature that we used was:
 
 ```c++
-ffi::Error RmsNormImpl(float eps, ffi::Buffer<ffi::DataType::F32> x,
-                       ffi::Result<ffi::Buffer<ffi::DataType::F32>> y)
+ffi::Error RmsNormImpl(float eps, ffi::Buffer<ffi::F32> x,
+                       ffi::Result<ffi::Buffer<ffi::F32>> y)
```

Comment:
There is also `ffi::ResultBuffer` now. Should we be using that here?

Reply:
Good suggestion! Done.